Wikipedia/Code/WMFLanguagesViewController.m

```diff
@@ -678,6 +683,13 @@ - (void)downloadArticlelanguages {
     [self.titleLanguageController
         fetchLanguagesWithSuccess:^{
             @strongify(self)
+                if (self.titleLanguageController.allLanguages.count == 0) {
+                    [self wmf_showEmptyViewOfType:WMFEmptyViewTypeNoOtherArticleLanguages theme:self.theme frame:self.view.bounds];
+                    self.hideLanguageFilter = YES;
```

Comment:
I could be missing a path it's mutated elsewhere, but since this is already set in the lifecycle method, we could just remove this line.